.github/workflows/ci.yml

```diff
@@ -20,3 +20,17 @@ jobs:
           coverage: xdebug
       - run: composer install
       - run: vendor/bin/phpunit --coverage-text
+
+  PHPStan:
+    name: PHPStan
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php:
+          - 8.1
```

Comment:
This technically doesn't need a matrix yet, but this paves the way for future PHP versions (and 8.2 is around the corner).